src/commonMain/kotlin/exchange.dydx.abacus/calculator/TradeInputCalculator.kt

```diff
@@ -1027,6 +1039,11 @@ internal class TradeInputCalculator(
             } else {
                 options.safeSet("maxLeverage", null)
             }
+            if (parser.asBool(options["needsReduceOnly"]) == true) {
+                options.safeSet("reduceOnlyPromptStringKey", null)
+            } else {
+                options.safeSet("reduceOnlyPromptStringKey", reduceOnlyPromptFromTrade(trade))
+            }
```

Comment:
is this reversed? if `needsReduceOnly` is true, I'm thinking  `reduceOnlyPromptStringKey` should be non-nil so client knows what to display

Reply:
ah i see. Small suggestion then to rename to something like `reduceOnlyUnavailablePromptStringKey` but not strong feelings on that